### PR TITLE
New package: systemd-boot-241

### PR DESCRIPTION
**Pull request by maxice8**
@gottox @duncaen @Cogitri

### CODE REVIEW

**Comment by the-maldridge** [comment]
I personally am somewhat opposed to adding more bootloaders, but the template looks as well written as your others.

**Comment by Johnnynator** [comment]
You wanted a comment, here is one :D
```
=> systemd-boot-241_1: running do_build ...
ninja: error: unknown target 'src/boot/efi/linuxx86.efi.stub'
=> ERROR: systemd-boot-241_1: do_build: 'ninja $targets' exited with 1
=> ERROR:   in do_build() at srcpkgs/systemd-boot/template:41
The command "common/travis/build.sh $BOOTSTRAP $ARCH" exited with 1.
```

File: srcpkgs/systemd-boot/template [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+license="LGPL-2.1-or-later"
+homepage="https://www.freedesktop.org/wiki/Software/systemd/systemd-boot"
+distfiles="https://github.com/systemd/systemd/archive/v${version}.tar.gz"
+checksum=b2561a8e1d10a2c248253f0dda31a85dd6d69f2b54177de55e02cd1d2778316e
+
+case "$XBPS_TARGET_MACHINE" in
+	x86_64*) efi_arch=x64 ;;
+	armv*) efi_arch=arm ;;
+	aarch64*) efi_arch=aa64 ;;
+	i686*) efi_arch=x86 ;;
```
Comment (Johnnynator) [inline srcpkgs/systemd-boot/template]: add a `*) broken="efi_arch is not set";;` Or does it have a generic `src/boot/efi/linux.efi.stub` target for ninja?

Comment (maxice8) [reply srcpkgs/systemd-boot/template]: Just use the wrong name for the i686 arch mostlikely

**Comment by q66** [comment]
You should make this "broken" for any arch not explicitly mentioned, or alternatively `archs="x86_64 armv* aarch64 i686"`, EFI is not really present anywhere except on x86 and ARM.

**Comment by q66** [comment]
I looked in the buildsystem. The allowed names are `ia32`, `x64`, `arm`, and `aa64`, there isn't anything else

**Comment by maxice8** [comment]
fixed

**Comment by q66** [comment]
:shipit:

**Comment by maxice8** [comment]
Is there interest in it for musl ? i can problably import the elogind musl_missing.h

File: srcpkgs/systemd-boot/template (hunk 1 above)
Comment (Gottox) [reply srcpkgs/systemd-boot/template]: I don't think `broken` is right here, I would prefer `archs=` as unsupported platforms not the same as broken platforms.

**Comment by maxice8** [comment]
@the-maldridge ok to merge into the void

**Comment by maxice8** [comment]
merged

**Pull request closed without merging**